examples/getting-started-app/README_RUS.md

```diff
@@ -325,9 +357,25 @@ getting-started-app $ touch app/roles/storage.lua
         return true
     end
     ```
-
 1. Функция обновления счета:
 
+   ```lua
+   local function update_balance(balance, amount)
+        -- конвертируем строку с балансом в число
+        local balance_decimal = decnumber.tonumber(balance)
+        balance_decimal = balance_decimal + amount
+        if balance_decimal:isnan() then
```

Comment:
Don't you want to resolve this issue in the scope of this PR - https://github.com/tarantool/cartridge-cli/issues/201 ?

Reply:
No. Not yet